src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -641,8 +641,60 @@ available on the [Fuchsia devsite].
 
 ### Running the compiler test suite
 
-Running the Rust test suite on Fuchsia is [not currently supported], but work is
-underway to enable it.
+Pre-requisites for running the Rust test suite on Fuchsia are:
+1. Installation of `rustc` via "[Targeting Fuchsia with a compiler built from source](#targeting-fuchsia-with-a-compiler-built-from-source)", including `config-env.sh` and `config.toml`.
```

Comment:
Installation isn't needed, just the config setup. x.py will build the compiler to the extent needed to run the tests.